sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateFormatter.scala

```diff
@@ -33,15 +32,15 @@ class Iso8601DateFormatter(
 
   @transient
   private lazy val formatter = getOrCreateFormatter(pattern, locale)
-  private val UTC = ZoneId.of("UTC")
+  private val UTC = ZoneOffset.UTC
```

Comment:
Not sure it matters, but this creates a field that always points to the same static instance. Can you just reference ZoneOffset.UTC directly rather than declare this?

Reply:
+1